docs/get-started/getting-started-databricks.md

```diff
@@ -155,3 +155,32 @@ storage location using Databricks [cluster log
 delivery](https://docs.databricks.com/clusters/configure.html#cluster-log-delivery-1).  Enable this
 option before starting the cluster to capture the logs.
 
+## Limitations
```

Comment:
I think it would be good to add something at the very beginning of the getting started on data bricks doc rather than here.  We also have other text in here that mentioned AQE .  Perhaps also put in FAQ.  I think @jlowe  had some other ideas.

Reply:
Agree this shouldn't be hidden at the end of the document.  I also think the [FAQ entry for DPP](https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/docs/FAQ.md#is-dynamic-partition-pruning-dpp-supported) and the [FAQ entry for AQE](https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/docs/FAQ.md#is-adaptive-query-execution-aqe-supported) should be updated.

Reply:
@jlowe  @tgravescs  I just added lines to mention DPP/AQE is not supported on databricks in FAQ. How is now?

Reply:
Thanks for the FAQ entries, but this section is still hidden at the end of the doc which was the original issue raised by this comment.

Reply:
@jlowe  @sameerz  I put limitation section to top. How do you think now?